devtools/cli/src/main/java/io/quarkus/cli/CreateExtension.java

```diff
@@ -0,0 +1,134 @@
+package io.quarkus.cli;
+
+import io.quarkus.cli.core.BaseSubCommand;
+import io.quarkus.devtools.commands.CreateProject;
+import io.quarkus.devtools.commands.data.QuarkusCommandException;
+import io.quarkus.devtools.project.BuildTool;
+import io.quarkus.devtools.project.codegen.SourceType;
+import io.quarkus.platform.tools.config.QuarkusPlatformConfig;
+import org.apache.maven.plugin.MojoExecutionException;
+import picocli.CommandLine;
+
+import java.io.File;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.concurrent.Callable;
+
+import static io.quarkus.devtools.commands.CreateExtension.extractQuarkiverseExtensionId;
+import static io.quarkus.devtools.commands.CreateExtension.isQuarkiverseGroupId;
+import static org.apache.commons.lang3.StringUtils.isEmpty;
+
+@CommandLine.Command(name = "create-extension", sortOptions = false, usageHelpAutoWidth = true, mixinStandardHelpOptions = false, description = "Creates the base of a Quarkus extension in different layout depending of the options and environment.")
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

Reply:
Well, Hello robot. It's part of the Picocli api :)